components/cdc/src/old_value.rs

```diff
@@ -206,6 +216,67 @@ mod tests {
     use tikv::storage::kv::TestEngineBuilder;
     use tikv::storage::txn::tests::*;
 
+    #[test]
+    fn test_old_value_resize() {
+        let capacity = 1024;
+
+        let mut old_value_cache = OldValueCache::new(ReadableSize(capacity));
+        let value = (
+            OldValue::Value {
+                value: b"value".to_vec(),
+            },
+            None,
+        );
+        // This is how the memory usage is calculated when we insert.
+        let size = Key::from_raw(&0_usize.to_be_bytes()).as_encoded().len()
+            + value.0.size()
+            + std::mem::size_of::<Option<MutationType>>();
```

Comment:
It's better to calculate size from OldValueCacheSizePolicy.

```suggestion
        let mut size_calc = OldValueCacheSizePolicy::default();
        size_calc.on_insert(&Key::from_raw(&0_usize.to_be_bytes()), &(value, None));
        let size = size_calc.current();
```

Reply:
Thanks!